apps/emqx/src/emqx_passwd.erl

```diff
@@ -51,7 +51,12 @@ check_pass({PassHash, Salt, Password}, {salt, HashType}) ->
     check_pass(PassHash, emqx_passwd:hash(HashType, <<Salt/binary, Password/binary>>));
 check_pass({PassHash, Salt, Password}, {HashType, salt}) ->
     check_pass(PassHash, emqx_passwd:hash(HashType, <<Password/binary, Salt/binary>>));
-check_pass(PassHash, PassHash) -> ok;
+check_password(PassHash1, PassHash2) when size(PassHash1) =:= size(PassHash2) ->
```

Comment:
This code won't compile because the clause name is different.

Reply:
sorry , I did not open my ide, edit it on web directly.